### PR TITLE
Refactoring for better logging

### PR DESCRIPTION
**Pull request by bmw**
First part of #4443. There should be no behavioral changes here but this does the refactoring to better support the other changes I made in #4443.

### CODE REVIEW

File: certbot/log.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+# Logging format
+CLI_FMT = "%(message)s"
+FILE_FMT = "%(asctime)s:%(levelname)s:%(name)s:%(message)s"
+
+
+logger = logging.getLogger(__name__)
+
+
+def pre_arg_setup():
```
Comment (ohemorange) [inline certbot/log.py]: nit: `pre_arg_parse_setup` and `post_arg_parse_setup`?

File: certbot/colored_logging.py [hunk 2]
```diff
@@ -1,45 +0,0 @@
-"""A formatter and StreamHandler for colorizing logging output."""
-import logging
-import sys
-
-from certbot import util
-
-
-class StreamHandler(logging.StreamHandler):
```
Comment (ohemorange) [inline certbot/colored_logging.py]: Would you mind terribly if this stayed in its own file? We do like random bits of OOO sprinkled throughout, and it's nice for them to stay in their own files like a proper OOO paradigm.

Comment (bmw) [inline certbot/colored_logging.py]: In #4446, I define two more of these simple handler classes. Would you like to see them in their own file as well? If so, I'll make this change and fix up the other PR too.

Comment (ohemorange) [inline certbot/colored_logging.py]: oh then nope, this way is better.

**Pull request merged**